pkg/cmd/secret/set/set.go

```diff
@@ -60,6 +61,9 @@ func NewCmdSet(f *cmdutil.Factory, runF func(*SetOptions) error) *cobra.Command
 			Use file as secret value
 			$ gh secret set MYSECRET < file.json
 
+			Print encoded secret value
+			$ gh secret set MYSECRET --no-store
```

Comment:
It feels weird to have to specify the secret variable name even in the case when the value is not going to be stored. Maybe this is reason enough to move this functionality to be its own command rather than an extra flag in `secret set`? But please don't make this change yet— I would first like to hear more thoughts from people in my team and potentially other users about this. Please be patient until we reach a decision 🙇

Reply:
Gotcha